src/Xamarin.Android.Build.Tasks/Utilities/Aapt2Daemon.cs

```diff
@@ -162,21 +165,32 @@ private void Aapt2DaemonStart ()
 				// and we are using netstandard 2.0
 				//StandardInputEncoding = Encoding.UTF8,
 			};
-			// We need to FORCE the StandardInput to be UTF8 so we can use 
+			// We need to FORCE the StandardInput to be UTF8 so we can use
 			// accented characters. Also DONT INCLUDE A BOM!!
 			// otherwise aapt2 will try to interpret the BOM as an argument.
-			Process aapt2;
+			Process aapt2 = null;
 			lock (lockObject) {
 				Encoding current = Console.InputEncoding;
 				try {
-					Console.InputEncoding = new UTF8Encoding (false);
+					try {
+						Console.InputEncoding = new UTF8Encoding (false);
+					} catch (IOException ioEx) {
+						// For some reason we can not set the InputEncoding. If this happens we don't
+						// want to crash Visual Studio with an Unhandled Exception.
+						// The downside of ignoring this is paths with accented characters will cause problems.
```

Comment:
If we end up with this PR as intermediate fix until a complete solution is found, then when you get a chance, if you could whip up a little known issue that describes the symptom users would see when using an accented character, that would be perfect.

Fingers crossed that the complete solution is just over the horizon though.

Reply:
The known issue can go alongside the draft release note for this PR in a `Documentation/release-notes/4722.md` file as part of this PR, so something like:

```
### IDE compatibility

- [Developer Community 1038779](https://developercommunity.visualstudio.com/content/problem/1038779/visual-studio-crash-down.html):
  ...

### Known issues

- [GitHub nnnn]():
  ... prevents building successfully if project paths contain accented
  characters or other non-ASCII UTF8 characters for projects configured to use
  AAPT2.
```

Reply:
@brendanzagaeski I can't get the actual error they might see as we have a bug stopping me from building in a directory with accented characters